data/en/1_0102.json

```diff
@@ -319,7 +319,7 @@
                         "type": "Checkbox"
                     }],
 
-                    "description": "Select all that apply",
+                    "description": "",
```

Comment:
These should fail validation but surprisingly don't.
if we don't have a value we should remove the description tag

Reply:
yeah I missed that one

Reply:
done but there are 277 other instances of this across the schemas.